### ISSUE TITLE
Players loading chunks in fabric is causing random lockups

### ISSUE THREAD

**Issue by MissPotato** [ChiselsAndBits/Chisels-and-Bits#1078]
Seemingly randomly my server would lock up until Watchdog kicked in. Once Watchdog killed the server it would dump a trace with chisel and bits in it. Said log is linked below. I reached out to theModded Discord who were telling me to not trust the watchdog, and to disable watchdog temporarily and instead use Spark to find the source instead. Unfortunately, Spark wasn't usable, as the locked up state prevented Spark from receiving any commands and the live viewer gives up without user input after 5 minutes. I'm going to make a feature request for them about this, but that's not relevant. 

It seems like whenever one of my player's bases, the only one with bits in them, gets loaded the server would freeze and the tick it was stuck on would never end. I even tried leaving it for 12 hours and it never recovered.  After disabling Chisel And Bits the crash hasn't happened again in 72 hours.

Long story short, I updated Chisel and Bits from 126 to 129, started getting deadlocks, removed Chisel And Bits and haven't had the dead lock again. 

Sorry, but the latest.log for this crash has been accidentally deleted, below is the latest log of the freshest boot in case it is still useful. https://mclo.gs/SW5yIxH

Watchdog: https://mclo.gs/b9Px7uJ
Fabric: 0.14.14
Minecraft: 1.19.2
Fabric API: 0.73.2
Other mods: https://pastebin.com/4XXSxFsW
Chisel And Bits Version: 1.3.9-129
OS/Eviroment: Linux Ubuntu VPS 20.04.5 LTS
Java: OpenJDK Runtime Environment (build 17.0.5+8-Ubuntu-2ubuntu120.04

**Comment by Neocentric** [comment]
I am having this same issue, but on forge in my single player world.  I am also using Chisel and Bits 1.29 on MC 1.19.2.

Here are my logs just in case: https://mclo.gs/Y19cu5I

**Comment by marchermans** [comment]
This is caused by porting lib, trying to load chunks which are currently being loaded.
Report it to them.

**Comment by Neocentric** [comment]
> This is caused by porting lib, trying to load chunks which are currently being loaded. Report it to them.

My modpack does not have porting lib installed.
